--- Library/PackageCache/com.unity.xr.arfoundation@4.1.3/Runtime/AR/ARFace.cs	2021-01-22 00:29:34.000000000 +0100
+++ Tools/ARFace.cs	2021-02-12 23:11:19.000000000 +0100
@@ -99,6 +99,10 @@
         // Creates an alias to the same array, but the caller cannot Dispose it.
         unsafe NativeArray<T> GetUndisposable<T>(NativeArray<T> disposable) where T : struct
         {
+#if UNITY_EDITOR
+            return disposable;
+#endif
+
             if (!disposable.IsCreated)
                 return default(NativeArray<T>);
 
