--- Library/PackageCache/com.unity.xr.arfoundation@4.0.0-preview.1/Runtime/AR/ARFace.cs	2020-03-25 21:55:49.000000000 +0100
+++ Tools/ARFace.cs	2020-04-16 10:44:08.000000000 +0200
@@ -99,6 +99,10 @@
         // Creates an alias to the same array, but the caller cannot Dispose it.
         unsafe NativeArray<T> GetUndisposable<T>(NativeArray<T> disposable) where T : struct
         {
+#if UNITY_EDITOR
+            return disposable;
+#endif
+
             if (!disposable.IsCreated)
                 return default(NativeArray<T>);
 
