--- Library/PackageCache/jp.keijiro.klak.ndi@0.2.4/Editor/PbxModifier.cs	2019-11-19 08:02:00.000000000 +0100
+++ Tools/PbxModifier.cs	2020-04-16 10:52:48.000000000 +0200
@@ -23,7 +23,11 @@
                 PBXProject proj = new PBXProject();
                 proj.ReadFromString(File.ReadAllText(projPath));
 
+#if UNITY_2019_3_OR_NEWER
+                string target = proj.GetUnityFrameworkTargetGuid();
+#else
                 string target = proj.TargetGuidByName("Unity-iPhone");
+#endif
 
                 // Add the header/library search path.
                 proj.AddBuildProperty(target, "HEADER_SEARCH_PATHS", "/NewTek\\ NDI\\ SDK/include");
